src/components/Sidebar/Sidebar.js

```diff
@@ -1,12 +1,16 @@
 import "./Sidebar.css";
 import {Link} from "react-router-dom";
+import {useData} from "../../contexts";
 
 
 function Sidebar(){
+
+    const {videosDispatch} = useData();
+
     return(
         <div className="sidebar">
             <Link className="nav-link" to="/">Home</Link>
-            <Link className="nav-link" to="/videos">Explore</Link>
+            <Link onClick={()=>videosDispatch({type:"FILTER_ALL"})} className="nav-link" to="/videos">Explore</Link>
```

Comment:
Suggestion: you can put your default filter as "All". So this step can be removed